optuna/visualization/_param_importances.py

```diff
@@ -122,56 +120,72 @@ def objective(trial):
         target:
             A function to specify the value to display. If it is :obj:`None` and ``study`` is being
             used for single-objective optimization, the objective values are plotted.
+            For multi-objective optimization, all objectives will be plotted if ``target``
+            is :obj:`None`.
 
             .. note::
-                Specify this argument if ``study`` is being used for multi-objective
-                optimization. For example, to get the hyperparameter importance of the first
-                objective, use ``target=lambda t: t.values[0]`` for the target parameter.
+                This argument can be used to specify which objective to plot if ``study``is being
```

Comment:
Seems like we need a blank here. ([ref](https://optuna--4871.org.readthedocs.build/en/4871/reference/visualization/generated/optuna.visualization.plot_param_importances.html#optuna.visualization.plot_param_importances:~:text=is%20None.-,Note,-This%20argument%20can))
```suggestion
                This argument can be used to specify which objective to plot if ``study`` is being
```